jib-core/src/main/java/com/google/cloud/tools/jib/api/JavaContainerBuilder.java

```diff
@@ -200,18 +191,14 @@ public JavaContainerBuilder addDependencies(List<Path> dependencyFiles) throws I
             .map(Entry::getKey)
             .collect(Collectors.toList());
     for (Path file : dependencyFiles) {
-      layerConfigurationsBuilder.addFile(
-          file.getFileName().toString().contains("SNAPSHOT")
-              ? LayerType.SNAPSHOT_DEPENDENCIES
-              : LayerType.DEPENDENCIES,
-          file,
-          dependenciesPath.resolve(
+      addedDependencies.add(
+          Paths.get(
               duplicates.contains(file.getFileName().toString())
                   ? file.getFileName().toString().replaceFirst("\\.jar$", "-" + Files.size(file))
```

Comment:
I think this new approach will fail because of this. Later, you will try to fetch the actual files by this name, which won't exist.